core/src/main/java/org/axonframework/eventhandling/AbstractEventProcessor.java

```diff
@@ -92,6 +93,11 @@ public Registration registerInterceptor(MessageHandlerInterceptor<? super EventM
         return () -> interceptors.remove(interceptor);
     }
 
+    @Override
+    public List<MessageHandlerInterceptor<? super EventMessage<?>>> getHandlerInterceptors() {
+        return interceptors;
```

Comment:
Please wrap in an Collections.unmodifiableList to prevent external modification to the list's contents.